src/components/RollupProductDevDoc.tsx

```diff
@@ -14,51 +11,45 @@ export type RollupProductDevDocProps = {
 
 const RollupProductDevDoc = ({ rollupType }: RollupProductDevDocProps) => {
   return (
-    <Box>
+    <div>
       {rollups[rollupType].map(
         ({ name, noteKey, website, developerDocs, l2beat }, idx) => {
           return (
-            <Flex key={idx} my={4} background="rollupDevDocList">
-              <Box p={4} pb={0}>
-                <Box>
-                  <Heading
-                    as="h4"
-                    fontSize={{ base: "md", md: "xl" }}
-                    fontWeight="500"
-                    lineHeight="1.4"
-                    my={4}
-                  >
+            <div key={idx} className="bg-rollupDevDocList my-4">
+              <div className="p-4 pb-0">
+                <div>
+                  <h4 className="my-4 text-md font-medium leading-relaxed md:text-xl">
                     {name}
-                  </Heading>
+                  </h4>
                   {noteKey.length > 0 && (
-                    <Text>
+                    <p>
                       * <Translation id={`page-layer-2:${noteKey}`} />
-                    </Text>
+                    </p>
                   )}
-                  <UnorderedList>
-                    <ListItem>
+                  <ul className="list-disc pl-5">
```

Comment:
Could we use to the shadcn components under https://github.com/ethereum/ethereum-org-website/blob/dev/src/components/ui/list.tsx ?

You should use the exported `UnorderedList` and `ListItem` in this case.

Reply:
so i need to make it back?

Reply:
done